src/styles/theme.ts

```diff
@@ -1,16 +1,15 @@
 const theme = {
   palette: {
     primary: {
-      main: "#f6f7fa",
-      hover: "#dee2ed",
+      main: "#7b3fe4",
+      dark: "#5a1cc3",
     },
     white: "#ffffff",
-    black: {
-      main: "#081132",
-    },
+    black: "#081132",
     grey: {
-      light: "#e2e5ee",
-      hover: "rgba(67, 85, 140, 0.1)",
+      light: "#f0f1f6",
+      main: "#e2e5ee",
+      dark: "#7c7e96",
     },
   },
   hoverTransition: "all 100ms",
```

Comment:
We should remove fontWeight from this file and cover all the settings in Typography

Reply:
Removed by the moment